### ISSUE TITLE
Optional pulse

### ISSUE THREAD

**Issue by ennvina** [ennvina/spellactivationoverlay#8]
All SAOs pulse by default, no questions asked.

However, some SAOs are not well fitted to this animation:
- Mage's Heating Up has no reason to pulse, only Hot Streak should pulse
- Shaman's Maelstrom Weapon has no reason to pulse with stacks 1-4, only the 5th stack should pulse

Pulsing is a strong invitation to take action "now", so it should pulse only when relevant.

Technically speaking, it could be as simple as adding a new `pulse` parameter to `SAO.ActivateOverlay`.

Be careful when the same spell ID is shared between pulsing and non-pulsing SAOs:
- when a spell ID has a non-pulsing SAO then a pulsing SAO, what happens?
- same question for the other way around (pulsing then non-pulsing)

As in #7 there is a possibility to use a fake "internal" spell ID. But this time it's a bit better:
- the weird show/hide isn’t a problem here because there’s already a signification change in animation
- in fact, this weird animation may even be desirable because it accentuates the "do something now"
- (the latter point works especially well when switching from non-pulsing to pulsing)
- (it _could_ be a problem when switching from pulsing to non-pulsing, except nothing does that now)

PS. this may be a slight change to the original Spell Activation Overlay which always pulsed.
In other words, this issue is probable in the #SomeChanges category.

**Comment by ennvina** [comment]
Fixed in #34
